lib/open_mic.rb

```diff
@@ -0,0 +1,30 @@
+class OpenMic
+  attr_reader :location, :date, :performers
+
+  def initialize(attributes)
+    @location = attributes[:location]
+    @date = attributes[:date]
+    @performers = []
+  end
+
+  def welcome(person)
```

Comment:
I would prefer to name this `user` because that reminds me this represents a `User` object.